apps/web/src/pages/upload.tsx

```diff
@@ -0,0 +1,217 @@
+      setPdf(url);
+      setPdfName(file.name);
+      setPdfFile(file);
+      setPdfSize((file.size / 1000000).toFixed(2));
+      setDisabled(false);
+    } catch (e) {
+      console.error(e);
+    }
+  };
+
+  /**
+   * Clear pdf information when a file "x" is clicked
+   */
+  const _handlePdfDelete = (e: MouseEvent) => {
+    e.preventDefault();
+    setPdf(null);
+    setPdfName(null);
+    setPdfFile(null);
+    setPdfSize(null);
+    setDisabled(true);
+  };
+
+  return (
+    <Box height="100vh" marginTop="36px">
+      <Flex position="absolute" margin="0px" zIndex={5000}>
+        <SideCreateForm curStep={1} />
+      </Flex>
+
+      <Heading
+        color="#2A2B2D"
+        fontSize="30px"
+        fontWeight={700}
+        lineHeight="38px"
+        marginLeft="36px"
+      >
+        Create form template
+      </Heading>
+      <Text
+        color="#4B4C4F"
+        fontSize="19px"
+        fontWeight={500}
+        lineHeight="26px"
+        pl="36px"
+      >
+        Upload your form PDF
+      </Text>
+      <Flex
+        /* outer white box */
+        padding="36px 24px 36px 24px"
+        flexDirection="column"
+        justifyContent={'center'}
+        alignItems="center"
+        gap="20px"
+        borderRadius="12px"
+        border="1px solid #E5E5E5"
+        height="auto"
+        margin="16px 36px 16px 36px"
+        backgroundColor="#FFF"
+        alignContent={'center'}
+      >
+        <Flex
+          /* dashed bordered box */
+          borderRadius="8px"
+          border="2px dashed #C3C7D1"
+          height="240px"
+          padding="40px 120px 48px 120px"
+          gap="16px"
+          flexDirection={'column'}
+          justifyContent={'center'}
+          alignItems={'center'}
+          backgroundColor={pdfName ? '#F1F7FF' : '#FFF'}
+          {...getRootProps()}
+        >
+          <UploadIcon
+            /* upload icon */
+            height="48px"
+            width="66px"
+          />
+          <Flex
+            /* all text below icon*/
+            flexDirection={'column'}
+            alignItems={'center'}
+          >
+            <Flex
+              /* drag file here or browse line */
+              gap="3px"
+            >
+              <Text fontWeight={500} size="16px" height="21px" align="center">
+                Drag file here or
+              </Text>
+              <label htmlFor="pdfInput">
+                <span>
+                  <Text
+                    textDecoration={'underline'}
+                    align="center"
+                    color="#1367EA"
+                  >
+                    browse
+                  </Text>
+                </span>
+              </label>
+            </Flex>
+            <Text
+              /* pdf qualifier */
+              fontSize="14px"
+              color="#808080"
+              fontWeight={500}
+            >
+              PDFs Only
+            </Text>
+          </Flex>
+          <input
+            type="file"
```

Comment:
You should also be able to use `inputFileRef` from `useBlob` here for upload to cloud storage at the end (Review step probably)